PyGame/main.py

```diff
@@ -0,0 +1 @@
+from package.Pygame.Pygame import pygame
```

Comment:
No. For better import use the next statement
```suggestion
import package.Pygame.Pygame
```

Reply:
After changes in you Pygame.py use the nest statements:
```suggestion
import package.Pygame.Pygame as p

p.Welcome()
```

Reply:
Also the bad practice is when your package name equals to the name of file inside the package.

The Pygame.py file should be renamed to the Greetings.py or something similar.